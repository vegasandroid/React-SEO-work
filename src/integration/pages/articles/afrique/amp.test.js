/**
 * @service afrique
 * @pathname /afrique/articles/c7yn6nznljdo
 */

import runGistTests from '../gistTests';

describe('AMP', () => {
  describe(pageType, () => {
    runGistTests();
  });
});
