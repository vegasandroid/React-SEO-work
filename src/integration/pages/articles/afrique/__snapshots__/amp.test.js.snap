// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AMP Articles Gist list item should match text 1`] = `"Lorem ipsum dolor sit amet, consectetur adipiscing elit."`;

exports[`AMP Articles Gist title should match text 1`] = `"Points cl√©s"`;
